addons/transform/org.openhab.transform.jinja/src/main/resources/readme.txt

```diff
@@ -0,0 +1 @@
+Bundle resources go in here!
```

Comment:
You don't need that.

Reply:
It was in copy template (javascript tranformer) :-)